frontend/src/pages/articles/components/article-card/article-card.tsx

```diff
@@ -162,12 +171,20 @@ const ArticleCard: React.FC<Properties> = ({
       </div>
       <div className={styles.footer}>
         <ul className={styles.reactions}>
-          <li>
-            <IconButton
-              iconName="comment"
-              className={styles.footerIcon}
-              label={MOCKED_REACTIONS.comments}
-            />
+          <li className={styles.reaction}>
+            <Link
+              to={{
+                pathname: articleRouteById as typeof AppRoute.ARTICLE,
+                hash: 'comments',
```

Comment:
Let's create constant for it and use for link hash and comments section anchor `id`

Reply:
done - [fa01c40](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/218/commits/fa01c40519b21cd920bcefc429a38963e44f6419)